automagicallyexecute/src/fetchJson.ts

```diff
@@ -0,0 +1,37 @@
+import fetch, {type BodyInit} from 'node-fetch'
```

Comment:
since you bumped to node 20: you could remove node-fetch. it's supported natively by the runtime

Reply:
meh, next time we do a rewrite ;)